wagtail/api/v2/utils.py

```diff
@@ -216,3 +216,20 @@ def parse_fields(fields_str, expect_close_bracket=False):
     fields, _ = parse_fields(fields_str)
 
     return fields
+
+
+def parse_boolean(value):
+    """
+    Parses strings into booleans using the following mapping (case-sensitive):
+
+    'true'   => True
+    'false'  => False
+    '1'      => True
+    '0'      => False
+    """
+    if value in ['true', '1']:
```

Comment:
I think it's fine as this would be used by developers. It'll be much easier to make this case-insensitive later on than it would be the other way anyway.